src/components/Modal.astro

```diff
@@ -0,0 +1,34 @@
+<div
+  id="modal-overlay"
+  class="flex justify-center items-center fixed inset-0 bg-black bg-opacity-50 z-10">
+  <div
+    class="bg-off-white dark:bg-gray-800 p-8 rounded-xl shadow-lg h-2/3 w-5/6 lg:w-1/2 md:h-1/2 flex flex-col overflow-y-auto">
+    <div class="flex justify-between items-start mb-6">
+      <h2 class="text-xl font-semibold">Welcome to LunaBot!</h2>
+      <button id="close-modal" class="cursor-pointer hover:text-luna-purple">
+        <svg
+          class="w-6 h-6"
+          fill="none"
+          stroke="currentColor"
+          viewBox="0 0 24 24"
+          xmlns="http://www.w3.org/2000/svg">
+          <path
+            stroke-linecap="round"
+            stroke-linejoin="round"
+            stroke-width="2"
+            d="M6 18L18 6M6 6l12 12"></path>
+        </svg>
+      </button>
+    </div>
+
+    <p>
+      Hi there! I’m LunaBot, your AI support companion. My goal is to help you
+      explore and practice Cognitive Behavioral Therapy techniques, guiding you
+      through exercises and offering support on anything you want to chat about. <br
+      /><br /> Please remember: While I’m designed to offer support and help you
+      develop skills based on CBT principles, I’m not a substitute for professional
+      therapy. For a comprehensive treatment plan, please consult a licensed mental
+      health professional.
+    </p>
```

Comment:
Does it work to break this into two `<p>` instead of the `<br />`? 
If not, the `<br />` is good!